grammar.js

```diff
@@ -278,6 +278,27 @@ module.exports = grammar(HTML, {
 
     assignment_expression: ($) =>
       seq(field('name', $.identifier), '=', field('value', $._any_expression)),
+    
+    // -------- ICU expressions ---------
+    icu_case: ($) => seq(
+      $.icu_category,
+      '{',
+      repeat1($._node),
+      '}'
+    ),
+    // icu_category: () => choice('=0', '=1', 'other'),
+    icu_category: () => /([=<>]\d+|other|[a-z][a-z0-9]+)/i,
+    icu_clause: () => choice('plural', 'select'),
+    icu: ($) =>
+      seq(
+        '{',
+        choice($._any_expression, $.concatenation_expression),
+        ',',
+        $.icu_clause,
+        ',',
+        repeat1($.icu_case),
+        '}',
+      ),
```

Comment:
```suggestion
    
    // -------- ICU expressions ---------
    icu: ($) =>
      seq(
        '{',
        choice($._any_expression, $.concatenation_expression),
        ',',
        $.icu_clause,
        ',',
        repeat1($.icu_case),
        '}',
      ),
      
    icu_clause: () => choice('plural', 'select'),

    icu_case: ($) => seq(
      $.icu_category,
      '{',
      repeat1($._node),
      '}'
    ),

    icu_category: () => /([=<>]\d+|other|[a-z][a-z0-9]+)/i,
```

Reply:
Can you explain the regex in a comment perhaps?

Reply:
instead of commenting I simplified the regex, I hope this is fine

Reply:
This regex allows too much I think. Perhaps just `/([=<>]\d+|[a-z]+)/i` would be better? This should cover the cases `=0`, `<0`, `>0`, `word`.

Reply:
I took a look into https://github.com/angular/angular/blob/edc1740d7e7f0296c719c7176c24031f55ae1487/packages/compiler/src/ml_parser/icu_ast_expander.ts and as far as I see, `>`, `<` aren't even allowed. But for `select`-clause I didn't see any restrictions

Reply:
I see. Then the regex is fine.